src/parser/shaman/elemental/modules/talents/EchoingShock.tsx

```diff
@@ -41,7 +41,7 @@ class EchoingShock extends Analyzer {
 
     this.casts += 1;
 
-    if (event.ability.guid !== SPELLS.LAVA_BURST.id) {
+    if (event.ability.guid !== SPELLS.LAVA_BURST.id && event.ability.guid !== SPELLS.EARTHQUAKE.id) {
```

Comment:
Im pretty sure this needs to be || not &&

Reply:
With ||, every cast would be a bad cast :D `&&` is correct